src/main/java/edu/kit/ipd/crowdcontrol/objectservice/event/EventLog.java

```diff
@@ -0,0 +1,79 @@
+package edu.kit.ipd.crowdcontrol.objectservice.event;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Can be used to create a stack of messages from a event subscriber.
+ * @author MarcelHollerbach
+ * @version 0.1
+ */
+public class EventLog {
+    private String name;
+    private List<LogEntry> logs;
+
+    /**
+     * Create a new Log
+     */
+    public EventLog() {
+        logs = new ArrayList<>();
+        name = "";
+    }
+
+    /**
+     * Get the name of the subscriber
+     *
+     * @return name of the subscriber
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Set the name of the subscriber
+     *
+     * The name will be later used to show which subscriber added the messages
+     *
+     * @param name name of the subscriber
+     */
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    /**
+     * Add a new message to this stack
+     * @param shortmessage a short explanation on what this message is about. No linebreaks allowed.
+     * @param longmessage detailed message what happends exactly. Linebreaks allowed.
+     */
+    public synchronized void addEntry(String shortmessage, String longmessage) {
+        logs.add(new LogEntry(shortmessage, longmessage));
```

Comment:
you can ignore my comment, but do we don`t really need a list, since you are using an index. For these concurrent things there is a ConcurrentLinkedQueue, which provides nonblocking FIFO operations